models/cow_protocol/tx_hash_labels/onramp/cow_protocol_tx_hash_labels_onramp_schema.yml

```diff
@@ -0,0 +1,64 @@
+version: 2
+
+models:
+  - name: cow_protocol_tx_hash_labels_onramp
+    meta:
+      blockchain: ethereum
+      sector: tx_hash_labels
+      category: onramp
+      contributors: gentrexha
+    config:
+      tags: ['labels', 'tx_hash', 'ethereum', 'onramp', 'dex']
+    description: "Trades that traded from one stable token to any other token"
+    columns:
+      - &blockchain
+        name: blockchain
+        description: "Blockchain"   
+      - &tx_hash
+        name: tx_hash
+        description: "Transaction hash of stable to stable trade"
+      - &name
+        name: name
+        description: "Label name"
+      - &category
+        name: category
+        description: "Label category"
+      - &contributor
+        name: contributor
+        description: "Wizard(s) contributing to labels"
+      - &source
+        name: source
+        description: "How were labels generated (query)"
+      - &created_at
+        name: created_at
+        description: "When were labels created"
+      - &updated_at
+        name: updated_at
+        description: "When were labels updated for the last time"
+      - &model_name
+        name: model_name
+        description: "Name of the label model sourced from"
+      - &label_type
+        name: label_type
+        description: "Type of label (see labels overall readme)"
+
+  - name: cow_protocol_tx_hash_labels_onramp_ethereum
+    meta:
+      blockchain: ethereum
+      sector: labels
+      category: onramp
+      contributors: gentrexha
+    config:
+      tags: ['labels', 'tx_hash', 'ethereum', 'onramp', 'dex']
+    description:  "Trades that traded from one stable token to any other token"
+    columns:
+      - *blockchain
+      - *tx_hash
+      - *name
+      - *category
+      - *contributor
+      - *source
+      - *created_at
+      - *updated_at
+      - *model_name
+      - *label_type
```

Comment:
Nit. New line at EOF (happening in many of these similar PRs). But not a huge deal.